HealthcareSystemFrontends/GraphicalEditor/Models/Drugs/Drug.cs

```diff
@@ -11,29 +11,17 @@ namespace GraphicalEditor.Models.Drugs
 {
     public class Drug
     {
-       
         public int DrugType_Id { get; set; }
-        public virtual DrugType DrugType { get; set; }
+        public DrugType DrugType { get; set; }
```

Comment:
Virtual was removed because DrugType isn't a table in the database, right?

Reply:
No,this class is in Model of front,and we use it just for initializing objects in database,so that we wont use that class from beckend